lms/djangoapps/verify_student/migrations/0005_auto_20180326_1437.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
```

Comment:
Nit: could you regenerate this migration with the `-n NAME` flag so the filename is more descriptive?